lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -302,6 +302,31 @@ static bool recursiveType(const StructType *ST, const Type *Ty) {
   return Run(Ty);
 }
 
+// Add decoration if needed
+void addFPBuiltinDecoration(SPIRVModule *BM, Instruction *Inst,
```

Comment:
Guess we need to remove the original function now https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/main/lib/SPIRV/SPIRVWriter.cpp#L3630 ?

Reply:
ah dang, nice catch

Reply:
fixed, thx